utils/packets/service.py

```diff
@@ -19,9 +19,32 @@ def ingest_survey_packet(survey_packet: str | os.PathLike):
         logger.info('Loaded survey packet')
         logger.info(f' -> Server ID: {metadata.server_id}')
         logger.info(f' -> License ID: {metadata.license_id}')
-        logger.info('\n')
+        logger.info('')
         logger.info(f' Total {df[UserSurveyFixedColumns.user_id.value].nunique()} unique users')
 
         # TODO: ingest in database
     except ValueError as e:
         raise ClickException(f'Error loading survey packet: {e}')
+
+
+def ingest_support_package(support_package: str | os.PathLike):
+    """
+    Load support package data and metadata.
+
+    :support_package: The path to the survey packet file.
+    """
+    try:
+        metadata, sp = load_support_package_file(support_package)
+        logger.info('Loaded survey packet')
```

Comment:
Both terms appear in discussions. Since [docs](https://docs.mattermost.com/manage/common-support-packet.html) use `support packet`, I'll change the naming to packet.

Reply:
Yes, they are confusing. One idea I thought was captured above (before your renaming) was to call the file - support package and the payload - support packet.